Vue/vaahtwo/stores/auth.js

```diff
@@ -129,6 +137,23 @@ export const useAuthStore = defineStore({
                 }
             }
         },
+        //-----------------------------------------------------------------------
+        signUp () {
+            let params = {
+                params: this.sign_up_items,
+                method: 'post'
+            };
+
+            vaah().ajax(
+                this.ajax_url+'/signup/post',
+                this.signUpAfter,
+                params
+            );
+        },
+        //---------------------------------------------------------------------
+        signUpAfter (data, res) {
+            console.log(data);
```

Comment:
@we-anil-y001 you should write meaningful code. Remove log code. 
Redirect to login page after successfully signup

Reply:
ok sir

Reply:
@we-vikram-wri231
Done redirect to login page after successfully.